### DISCUSSION TITLE
xx65 Közösségi Összesítő Nyilatkozat

### DISCUSSION THREAD

**Discussion by Macskafarka** [nav-gov-hu/eVAT#81] [Általános]
Kiváltja-e az eÁFA bevallás az xxA60 Közösségi Összesítő jelentést?
Ha igen, akkor kaphatnánk segítséget arra vonatkozóan, hogy az 1-4 lapokhoz mely Standard adókódok tartoznak?

**Comment by renced42** [comment]
Kedves @Macskafarka 

Nem váltja ki az eÁFA az A60-as nyilatkozatot jelenleg.
